paimon-e2e-tests/pom.xml

```diff
@@ -34,6 +34,7 @@ under the License.
     <properties>
         <flink.shaded.hadoop.version>2.8.3-10.0</flink.shaded.hadoop.version>
         <flink.sql.connector.kafka>flink-sql-connector-kafka</flink.sql.connector.kafka>
+        <flink.sql.connector.hive>flink-sql-connector-hive-2.3.9_${scala.binary.version}</flink.sql.connector.hive>
```

Comment:
Does the e2e tests need to add the test case for 3.x?

Reply:
I'd love to. However currently I can't find any Hive 3.x docker image. Let's create an issue for adding Hive 3.x e2e tests.